sensing/pointcloud_preprocessor/src/outlier_filter/radius_search_2d_outlier_filter_nodelet.cpp

```diff
@@ -28,8 +28,8 @@ RadiusSearch2DOutlierFilterComponent::RadiusSearch2DOutlierFilterComponent(
 {
   // set initial parameters
   {
-    min_neighbors_ = static_cast<size_t>(declare_parameter("min_neighbors", 5));
-    search_radius_ = static_cast<double>(declare_parameter("search_radius", 0.2));
+    min_neighbors_ = declare_parameter<uint16_t>("min_neighbors");
```

Comment:
```suggestion
    min_neighbors_ = declare_parameter<int>("min_neighbors");
```

Reply:
Could you also change the line 64 to
int k = kd_tree_->radiusSearch(i, search_radius_, k_indices, k_sqr_distances);

and also tests whether it works?
Thanks